evmd/go.mod

```diff
@@ -280,7 +280,7 @@ replace (
 	github.com/cosmos/evm => ../
 	github.com/cosmos/evm/evmd => ./evmd
 	// use Cosmos geth fork
-	github.com/ethereum/go-ethereum => github.com/cosmos/go-ethereum v1.15.11-cosmos-0
+	github.com/ethereum/go-ethereum => github.com/yihuang/go-ethereum v1.10.21-0.20250721081216-98795aa589d5
```

Comment:
Is there a reason for this change? I checked `github.com/yihuang/go-ethereum v1.10.21-0.20250721081216-98795aa589d5` and found that it is in the same state as `github.com/cosmos/go-ethereum v1.15.11-cosmos-0`.

Reply:
reverted this dependency change.